### ISSUE TITLE
To add a folder access password, add a. Password file in the folder to set the password

### ISSUE THREAD

**Issue by msdnos** [DirectoryLister/DirectoryLister#323]
To add a folder access password, add a. Password file in the folder to set the password

**Comment by PHLAK** [comment]
Thank you for the suggestion but password protected directories is not something we will be supporting. If you need this feature Directory Lister is not the right choice.

**Comment by Body4** [comment]
> Thank you for the suggestion but password protected directories is not something we will be supporting. If you need this feature Directory Lister is not the right choice.

Hi Phlak, I'm also interested for this functionality but I don't understand your answer. Why Directory Lister isn't the right choice? 
I saw the option 'hidden files' so, for me, you have already thought about this use case which is to 'protected a folder/file'. 

Just for my understanding, there is a particular problem to implement this kind of functionality ? Maybe I'm missing a best practice?

I tried to accomplish this with nginx but isn't very simple unless we have 2 separates vhost (easy way) and I don't want this.
That why, I think, that will be more simple and efficiency if Directory Lister did it. In my opinion..
